src/openrct2/world/Map.cpp

```diff
@@ -2054,12 +2054,12 @@ int32_t map_get_tile_quadrant(int32_t mapX, int32_t mapY)
  *
  *  rct2: 0x00693BFF
  */
-bool map_surface_is_blocked(int16_t x, int16_t y)
+bool map_surface_is_blocked(CoordsXY mapCoords)
 {
-    if (x >= 8192 || y >= 8192)
+    if (mapCoords.x >= 8192 || mapCoords.y >= 8192)
```

Comment:
While you're at it: this is a magic value of 256 × 32. We have a constant called MAX_MAP_SIZE_TECHNICAL or something like that with a value of 256. Could you modify the code to use it?

Reply:
There is a function that can be called instead. That checks for outside of range.

Reply:
Could you point me to the function, I couldn't find it :\

In the meantime, I changed it to use `MAXIMUM_MAP_SIZE_TECHNICAL` instead.

Reply:
I think it's `map_is_location_valid()`.

Reply:
Thanks!
Small note, it also checks if the coords are `>= 0` which wasn't before, but I guess this is a plus.

Reply:
Thanks!
Small note, it also checks if the coords are >= 0 which wasn't before, but I guess this is a plus.

Reply:
Previously, it didn't need to, because the input parameters couldn't be negative:
```
bool map_surface_is_blocked(int16_t x, int16_t y)
```
so previously, just checking for >=8192 was sufficient to cover both.